shared/test/showcase/echo_test.rb

```diff
@@ -20,11 +20,100 @@
 class EchoTest < ShowcaseTest
   def test_echo
     client = Google::Showcase::V1alpha3::Echo.new(
-      credentials: GRPC::Core::Channel.new(
-        "localhost:7469", nil, :this_channel_is_insecure))
+      credentials: GRPC::Core::Channel.new("localhost:7469", nil, :this_channel_is_insecure)
+    )
 
     response = client.echo content: 'hi there!'
 
     assert_equal 'hi there!', response.content
   end
+
+  def test_chat_closure
+    client = Google::Showcase::V1alpha3::Echo.new(
```

Comment:
sort of unrelated nit but can this move to a before/after (also need to shutdown the channel after each test)?

Reply:
Yes. Each test is run on a separate object and will have its own channel.